cas-server-core/src/main/java/org/jasig/cas/AbstractCentralAuthenticationService.java

```diff
@@ -150,6 +159,15 @@ protected final void doPublishEvent(final ApplicationEvent e) {
 
 
     /**
+     * Gets a synchronization handle.
+     * @param id String mutex id
+     * @return SimpleMutex
+     */
+    protected final SimpleMutex getMutex(final String id) {
```

Comment:
Which is held in the mutex, so it'll never go away... I'll correct that before I kill off the class.